projects/step-core/styles/components/_theme-config.scss

```
@@ -254,4 +254,17 @@ $light-primary-text: white;
     margin: 0;
     padding: 0;
   }
+
+  /**
+  This will create a nice scrollbar inside the form field when the chips content are very long.
+  Otherwise they will overflow the container.
+   */
+  .mat-chip-list-wrapper {
+    overflow: auto;
```

```suggestion
    overflow-y: auto;
```

I assume you want horizontal scrolling. I would always explicitly set it to prevent unintended scroll bars in teh wrong direction